app/controllers/idv_controller.rb

```diff
@@ -1,7 +1,30 @@
 class IdvController < ApplicationController
-  def p1
+  def register_user(args = nil)
+    agent = Proofer::Agent.new(vendor: :mock)
+    app_vars = params.slice(:first_name, :last_name, :dob)
+    applicant = Proofer::Applicant.new(app_vars)
+    session[:resolution] = agent.start(applicant)
+    session[:question_number] = 0
+    render 'question', locals: {quest: session[:resolution].questions[session[:question_number]]}
   end
```

Comment:
+1 to @monfresh on controller conventions. It's more Railsy, even if it feels more onerous at first to write.